### ISSUE TITLE
A list of builtin functions

### ISSUE THREAD

**Issue by boynedmaster** [PyHP-pph/PyHP-pph#19]
We can either take after PHP and have thousands of inconsistently named builtin functions or be like Python and have small amount of builtins with imports handling the rest. If we truly want this language to be shitty, I suggest taking the former option.

**Comment by alexbuzzbee** [comment]
I agree. We should take the PHP approach and put the entire standard library into the global namespace by default (possibly with no option to disable it).

**Comment by boynedmaster** [comment]
Perhaps we can use this issue as a place to suggest builtins until they are in a formal specification?

**Comment by ghost** [comment]
We should just have a bunch of specification files so the inconsistency can truly prevail.
